litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Steps/TuneTheQueries/EditPanelsWizard/QueryEditingWizard/index.tsx

```diff
@@ -189,6 +191,7 @@ const QueryEditingWizard: React.FC<QueryEditingWizardProps> = ({
             options={panelGroupsList}
             getOptionLabel={(option: Option) => option.name}
             style={{ width: '10.3rem' }}
```

Comment:
please remove inline styles

Reply:
we had to override single property so we have used in this way to keep the code base consistent (used similarly in workflows section) @Saranya-jena , cc: @Jonsy13 @amityt @S-ayanide